.github/ISSUE_TEMPLATE/flaky-e2e-spec-report.yml

```diff
@@ -12,24 +12,11 @@ body:
     attributes:
       label: Spec file
       placeholder: eg. wp-likes__post.js
-    validations:
-      required: true
   - type: input
-    id: suite
-    attributes:
-      label: Suite or step
-      placeholder: eg. Like new post
-  - type: dropdown
-    id: framework
+    id: teamcity_id
     attributes:
-      label: Framework
-      description: Which framework does the spec belong to?
-      options:
-        - Selenium
-        - Playwright
-      multiple: true
-    validations:
-      required: true
+      label: TeamCity ID
+      placeholder: eg. 648729
```

Comment:
Small suggestions, but maybe we can add a description clarifying this a bit more that...
1. We're looking for the TeamCity ID of a _build_ that includes the test failing.
2. Should we tell people they can find this at the end of the URI when the click on a TeamCity build?

Reply:
That's a good point. This field is just a transitory one while I try to get a basic TamperMonkey one working too.